claude_coordinator.py

```diff
@@ -169,7 +179,18 @@ def run(self):
 
 if __name__ == '__main__':
     num_workers = int(sys.argv[1]) if len(sys.argv) > 1 else 4
-    db_path = sys.argv[2] if len(sys.argv) > 2 else "claude_tasks.db"
+
+    # Determine database path with proper precedence:
+    # 1. Explicit argument (highest priority)
+    # 2. KLAUSS_DB_PATH environment variable
+    # 3. Default to "claude_tasks.db"
+    if len(sys.argv) > 2:
+        db_path = sys.argv[2]
+    elif 'KLAUSS_DB_PATH' in os.environ:
```

Comment:
Why do we need an environment variable for this? Couldn't this just come from config?